inmap/cmd/inmap.go

```diff
@@ -64,17 +65,34 @@ var DefaultScienceFuncs = []inmap.CellManipulator{
 // runtime, and cleanup, respectively.
 func Run(dynamic, createGrid bool, scienceFuncs []inmap.CellManipulator, addInit, addRun, addCleanup []inmap.DomainManipulator) error {
 
+	startTime := time.Now()
+
 	// Start a function to receive and print log messages.
+	logfile, err := os.Create(Config.Logfile)
+	if err != nil {
+		log.Fatal("Cannot create file", err)
```

Comment:
In general, we want to add context to errors and return them instead of killing the program. Here: 
```return fmt.Errorf("inmap: creating log file: %v", err)```